apps/spruce/src/pages/adminSettings/tabs/AuthenticationTab/AuthenticationTab.tsx

```diff
@@ -0,0 +1,6 @@
+export const AuthenticationTab = () => (
+  <div className="flex flex-col gap-4">
+    <h1 className="text-xl font-bold">Authentication</h1>
```

Comment:
same comment

also, delete classNames
```suggestion
  <div>
    <h1>Authentication</h1>
```